lib/common/output_log.c

```diff
@@ -94,35 +130,31 @@ log_version(pcmk__output_t *out, bool extended) {
     priv = out->priv;
 
     if (extended) {
-        do_crm_log(priv->log_level, "Pacemaker %s (Build: %s): %s",
```

Comment:
Technically the behavior will be different now if a logger output object's log level is set to `LOG_STDOUT`... which I believe is possible, but would never make any sense to do. I'm fine with ignoring since `pcmk__output_t` is internal anyway.

Reply:
Yep, I considered failing or asserting if someone tries to set a logger object to STDOUT, but decided it wasn't worth bothering with